CHANGELOG.md

```diff
@@ -8,41 +8,21 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 ## Hammer-5
 
 ### Added
-- Use conversion_hosts API for RHV target cluster warnings instead of host CF tags [(#847)](https://github.com/ManageIQ/manageiq-v2v/pull/847)
```

Comment:
This one can stay in, it's not related to the new Conversion Host Enablement settings. I think it's the only conversion host related PR that should stay.